libyul/backends/wasm/WasmCodeTransform.cpp

```diff
@@ -140,11 +140,12 @@ wasm::Expression WasmCodeTransform::operator()(FunctionCall const& _call)
 				m_functionsToImport[builtin->name] = std::move(imp);
 			}
 		}
-		else if (builtin->literalArguments && contains(builtin->literalArguments.value(), true))
+		// TODO: improve contains or introduce not_contains(.., std::nullopt)
+		else if (builtin->literalArguments)
```

Comment:
This is the only one that needs the helper, isn't it? All the other places really only want to know if the i-th argument is a literal, not if there are any literal arguments, or am I blind?

Reply:
This line here I think also only needs it as an optimisation and nothing else.